app/index.html

```diff
@@ -69,8 +69,8 @@
                             <li ng-class="{current: isCurrent('/')}"><a href="#"><i class="glyphicon glyphicon-home"></i> Dashboard</a></li>
                             <li ng-class="{current: isCurrent('/transactions')}"><a href="#/transactions"><i class="glyphicon glyphicon-list-alt"></i> Transaction log</a></li>
                             <li ng-show="userGroupAdmin" ng-class="{current: isCurrent('/audits')}"><a href="#/audits"><i class="glyphicon glyphicon-book"></i> Audit Log</a></li>
-                            <li ng-show="userGroupAdmin" ng-class="{current: isCurrent('/clients')}"><a href="#/clients"><i class="glyphicon glyphicon-th"></i> Clients</a></li>
                             <li ng-show="userGroupAdmin" ng-class="{current: isCurrent('/channels')}"><a href="#/channels"><i class="glyphicon glyphicon-random"></i> Channels</a></li>
+                            <li ng-show="userGroupAdmin" ng-class="{current: isCurrent('/clients')}"><a href="#/clients"><i class="glyphicon glyphicon-th"></i> Clients</a></li>
```

Comment:
I'm not sure why the order has been changed, perhaps this is a refactor mistake?